package.json

```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://github.com/schelling-network/contracts#readme",
   "dependencies": {
+    "@openzeppelin/contracts": "^4.0.0",
     "@razor-network/merkle": "^0.6.0",
     "jsonfile": "^6.1.0",
     "openzeppelin-solidity": "^3.1.0"
```

Comment:
Can we remove `openzeppelin-solidity` dependency since we are not using it anymore? @adi44

Reply:
I have removed it from the package.json @dev1644